kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -191,52 +252,60 @@ define([
                 },
             });
 
-            // Just pass the update along to the input widget.
-            paramsBus.listen({
-                key: {
-                    type: 'update',
-                    parameter: parameterSpec.id,
-                },
-                handle: function (message) {
-                    fieldWidget.bus.emit('update', {
-                        value: message.value,
-                    });
-                },
-            });
-
             return fieldWidget;
         }
 
         function updateRowNumbers(filePathRows) {
             filePathRows.forEach((filePathRow, index) => {
-                $(filePathRow)
-                    .find(`.${cssBaseClass}__file_number`)
-                    .text(index + 1);
+                filePathRow.querySelector(`.${cssBaseClass}__file_number`).textContent = index + 1;
             });
         }
 
-        function addRow(e) {
-            $(e.target)
-                .prev('table')
-                .append(
-                    tr({
-                        class: `${cssBaseClass}__table_row`,
-                        dataElement: `${cssClassType}-fields-row`,
-                    })
-                );
+        /**
+         * Responsibilities
+         *  - make row id
+         *  - make initial table row layout
+         *  - set up row parameters
+         *  - make the row
+         * @param {*} params
+         */
+        function addRow(params) {
+            if (!params) {
+                // initialize params
+                params = Object.assign({}, model.getItem('defaultParams'));
```

Comment:
Without doing a copy this way, the kept defaults can get overwritten. IMO, a bug in the Props module, but this gets around it.